pynsist.cfg

```diff
@@ -42,18 +42,70 @@ pypi_wheels = PyQt5==5.14.2
     pywin32==227
     send2trash==1.7.1
 files = share/
-exclude = pkgs/PyQt5/QtBluetooth.pyd
-    pkgs/PyQt5/QtDesigner.pyd
-    pkgs/PyQt5/QtQuick.pyd
-    pkgs/PyQt5/QtQuickWidgets.pyd
-    pkgs/PyQt5/QtQml.pyd
-    pkgs/PyQt5/Qt/bin/Qt5Bluetooth.dll
+exclude = pkgs/PyQt5/Qt/bin/Qt5Bluetooth.dll
+    pkgs/PyQt5/Qt/bin/Qt5DBus.dll
     pkgs/PyQt5/Qt/bin/Qt5Designer.dll
+    pkgs/PyQt5/Qt/bin/Qt5Help.dll
+    pkgs/PyQt5/Qt/bin/Qt5Location.dll
+    pkgs/PyQt5/Qt/bin/Qt5Multimedia.dll
+    pkgs/PyQt5/Qt/bin/Qt5MultimediaWidgets.dll
+    pkgs/PyQt5/Qt/bin/Qt5NetworkAuth.dll
+    pkgs/PyQt5/Qt/bin/Qt5Nfc.dll
+    pkgs/PyQt5/Qt/bin/Qt5OpenGL.dll
+    pkgs/PyQt5/Qt/bin/Qt5Positioning.dll
+    pkgs/PyQt5/Qt/bin/Qt5PositioningQuick.dll
+    pkgs/PyQt5/Qt/bin/Qt5PrintSupport.dll
+    pkgs/PyQt5/Qt/bin/Qt5Qml.dll
+    pkgs/PyQt5/Qt/bin/Qt5QmlModels.dll
+    pkgs/PyQt5/Qt/bin/Qt5QmlWorkerScript.dll
     pkgs/PyQt5/Qt/bin/Qt5Quick.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickControls2.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickParticles.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickShapes.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickTemplates.dll
+    pkgs/PyQt5/Qt/bin/Qt5QuickTemplates2.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickTest.dll
     pkgs/PyQt5/Qt/bin/Qt5QuickWidgets.dll
-    pkgs/PyQt5/Qt/bin/Qt5Qml.dll
+    pkgs/PyQt5/Qt/bin/Qt5RemoteObjects.dll
+    pkgs/PyQt5/Qt/bin/Qt5Sensors.dll
+    pkgs/PyQt5/Qt/bin/Qt5SerialPort.dll
+    pkgs/PyQt5/Qt/bin/Qt5Sql.dll
+    pkgs/PyQt5/Qt/bin/Qt5Test.dll
+    pkgs/PyQt5/Qt/bin/Qt5WebChannel.dll
+    pkgs/PyQt5/Qt/bin/Qt5WebSockets.dll
+    pkgs/PyQt5/Qt/bin/Qt5WinExtras.dll
+    pkgs/PyQt5/Qt/bin/Qt5Xml.dll
+    pkgs/PyQt5/Qt/bin/Qt5XmlPatterns.dll
+    pkgs/PyQt5/Qt/bin/concrt140.dll
+    pkgs/PyQt5/Qt/bin/d3dcompiler_47.dll
+    pkgs/PyQt5/Qt/bin/libEGL.dll
+    pkgs/PyQt5/Qt/bin/libGLESv2.dll
+    pkgs/PyQt5/Qt/bin/libeay32.dll
+    pkgs/PyQt5/Qt/bin/msvcp140.dll
+    pkgs/PyQt5/Qt/bin/opengl32sw.dll
+    pkgs/PyQt5/Qt/bin/ssleay32.dll
```

Comment:
i'm paranoid so libEGL.dll, libGLESv2.dll, ssleay32.dll and opengl32sw.dll might be worth keeping.